pkg/xen-tools/Dockerfile.in

```diff
@@ -74,6 +74,7 @@ RUN apk add --no-cache \
     pixman=0.34.0-r5   \
     yajl=2.1.0-r0      \
     keyutils=1.5.10-r0 \
+    screen=4.6.2-r1    \
```

Comment:
Lets not pull screen in for now -- anyone debugging the system can always apk add screen (and in fact, we should probably move towards using a socket for stdio anyway -- but that shouldn't be part of this PR).

Reply:
Note that you don't have to be in the same R/O rootfs/container for using screen

Reply:
Hi @rvs,  excluded this change to pull in screen. I've updated the PR. Please take a look.